### ISSUE TITLE
Login redirect not working correctly behind ingress

### ISSUE THREAD

**Issue by skatsaounis** [canonical/netbox#40]
### Bug Description

When integrated with Traefik charm, when a user tries to access a page that requires authentication, the path prefix is not added, leading to 404:

### To Reproduce

Follow the instructions: `https://discourse.charmhub.io/t/netbox-docs-getting-started/13760` and then try to access from web as unathenticated user, e.g., the rack elevations:

`http://k8s.fqdn/maas-model-netbox/dcim/rack-elevations/`
--->
`http://k8s.fqdn/login/?next=/maas-model-netbox/dcim/rack-elevations/`
instead of
`http://k8s.fqdn/maas-model-netbox/login/?next=/maas-model-netbox/dcim/rack-elevations/`

### Environment

|App     |        Version | Status | Scale|  Charm    |       Channel       | Rev |
| -- | --- | --- | --- | --- | -- | --- |
|netbox      |         |    active    |  1  |netbox     |     latest/stable |  21|
|postgresql-k8s | 14.12  |  active   |   1 | postgresql-k8s|  14/stable  |    444|
|redis-k8s   |    7.2.5  |  active   |   1 | redis-k8s   |    latest/edge  |   38|
|s3-integrator   |       |  active  |    1 | s3-integrator |  latest/edge  |   79|

juju version 3.5.3

### Relevant log output

```shell
N/A
```

**Comment by skatsaounis** [comment]
It seems that `BASE_PATH` should be set accordingly: https://netboxlabs.com/docs/netbox/en/stable/configuration/system/#base_path

**Comment by javierdelapuente** [comment]
Thanks for the report @skatsaounis 

I am checking it now, with Traefik Rev 219, and when using the url from `juju run traefik-k8s/0 show-proxied-endpoints --format=yaml`, as you say, clicking on for example Racks redirect to `http://.../None`, however the top link "login" works.

Also adding a final `/` to the url, (like `http://netbox_hostname/netbox-tutorial-netbox/ ` instead of `http://netbox_hostname/netbox-tutorial-netbox`), it doest not redirect to the root location. 

Can you confirm that the same happens to you so we tackle that problem?

Thanks

**Comment by javierdelapuente** [comment]
Thank you for the report. The None maybe related to the upstream project.

I have added the [jira bug](https://warthogs.atlassian.net/browse/ISD-2787) for it, so at least we review the `/`.

**Comment by alithethird** [comment]
@javierdelapuente is this still relevant?

**Comment by javierdelapuente** [comment]
@alithethird yes it is. It will be tackled in the next iteration in the NetBox charm

**Comment by alithethird** [comment]
The `/` issue is fixed with the new charm https://github.com/canonical/netbox-k8s

https://charmhub.io/netbox-k8s
